### ISSUE TITLE
The 'i' / info button on the bag frame does nothing any more.

### ISSUE THREAD

**Issue by arcadepro** [xod-wow/LiteBag#22]
It used to open the the options config before the ace config change I think.

Also throws the error:
````
18x [string "*ContainerPanel.xml:17_OnClick"]:1: attempt to index global 'LB' (a nil value)
[string "*ContainerPanel.xml:17_OnClick"]:1: in function <[string "*ContainerPanel.xml:17_OnClick"]:1>

Locals:
self = LiteBagBackpackPanelInfoButton {
 0 = <userdata>
 texture = LiteBagBackpackPanelInfoButtonTexture {
 }
}
button = "LeftButton"
down = false
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index global 'LB' (a nil value)"
````

**Comment by mbattersby** [comment]
Duplicate of #20.

This is fixed in 10.0.1 but CurseForge are refusing to publish it.

**Comment by mbattersby** [comment]
Curse have published 10.0.2 despite still refusing to release 10.0.1. Ugh.,
